test/test_torch.py

```diff
@@ -19310,6 +19310,12 @@ def _do_transpose(self, x, contiguous=False, dim0=0, dim1=1):
         else:
             return x.transpose(dim0, dim1)
 
+    @dtypes(*(torch.testing.get_all_int_dtypes() + torch.testing.get_all_fp_dtypes()))
+    def test_conj_self(self, device, dtype):
+        t = torch.ones(5, 5, device=device)
+        s = t.fast_conj()
+        self.assertTrue(s is t)
```

Comment:
do we actually want this or do we want `s` to be a view of `t`?

Reply:
we could but I think we should just return same tensor as input with same storage to make it a complete no op